.github/workflows/tools/update-issues/issues.js

```diff
@@ -0,0 +1,539 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A set of classes to represent GitHub issues & comments.
+ */
+
+const github = require('@actions/github');
+const core = require('@actions/core');
+
+const octokit = github.getOctokit(process.env.GITHUB_TOKEN);
+const [owner, repo] = process.env.GITHUB_REPOSITORY.split('/');
```

Comment:
These should probably be all-caps since they're constants even if they're dynamic.

Reply:
I prefer to keep them as-is, since it means I can use this syntax later in the queries:

```js
{
  owner,
  repo,
}
```

Instead of:

```js
{
  owner: OWNER,
  repo: REPO,
}
```